wgsl/index.bs

```diff
@@ -5361,11 +5373,10 @@ function_decl
   : attribute_list* function_header compound_statement
 
 function_header
-  : FN IDENT PAREN_LEFT param_list PAREN_RIGHT function_return_type_decl_optional
+  : FN IDENT PAREN_LEFT param_list PAREN_RIGHT function_return_type_decl?
```

Comment:
this is technically unrelated, just a drive-by thing
let me know if you want this to be excluded for purity of the PR